packages/contracts/test/contracts/unichain/DataUnionTemplate.test.ts

```diff
@@ -59,13 +60,15 @@ describe("DataUnionTemplate", () => {
     const o = others.map(outsider => outsider.address)
 
     let testToken: TestToken
+    let feeOracle: Contract
     let dataUnionSidechain: DataUnionTemplate
     let dataUnionSidechainAgent: DataUnionTemplate
     let dataUnionSidechainMember0: DataUnionTemplate
 
     before(async () => {
         testToken = await deployContract(creator, TestTokenJson, ["name", "symbol"]) as TestToken
         await testToken.mint(creator.address, parseEther("10000"))
+        feeOracle = await deployContract(creator, FeeOracleJson, [parseEther("0.1")])
```

Comment:
Now both adminFee and protocolFee are set to 0.1, meaning the test wouldn't catch bugs where the two are accidentally swapped. This can be solved by setting the protocol fee to another value.

Reply:
changed to 9% and 1%; that's maybe more realistic, too.